src/WinGet.RestSource/WinGet.RestSource.csproj

```diff
@@ -1,7 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netstandard2.0</TargetFramework>
+    <TargetFramework>netstandard2.1</TargetFramework>
```

Comment:
To facilitate movement of common code to utilities libraries, netcoreapp3.1 makes it more straightforward. Is there any reason to keep this particular library netstandard?

Reply:
Not that I'm aware of, I'm happy to go to .NET Core. Along those lines, any reason we shouldn't go straight to 5.0?